Makefile

```diff
@@ -73,7 +73,7 @@ rpm-release: srpm-release
 	mock -r $(MOCK_CONFIG) --resultdir BUILD/RPM -D "rel_build 1" --rebuild BUILD/SRPM/python-aexpect-$(VERSION)-*.src.rpm
 
 check: clean
-	inspekt checkall --disable-lint W1618
+	inspekt checkall --disable-lint W1618,R0205
```

Comment:
OK, we do need to support py2 compatibility so we have to explicitly use `object`.